src/main/java/net/minecraftforge/fml/common/gameevent/PlayerEvent.java

```diff
@@ -38,11 +38,13 @@ private PlayerEvent(EntityPlayer player)
     	/**
     	 * This entity is a clone of what the player picked up, containing intact ItemStack information.
     	 */
-        public final EntityItem pickedUp;
-        public ItemPickupEvent(EntityPlayer player, EntityItem pickedUp)
+        public final EntityItem originalEntity;
+        public final ItemStack pickedUp;
```

Comment:
getters

Reply:
Can-do. I will note that...
ItemCraftedEvent, ItemSmeltedEvent, and PlayerChangedDimensionEvent don't use getters.
PlayerRespawnEvent is the only one that does.
And that's just within PlayerEvent (which also does not use a getter for the player).

Reply:
Yes, a lot of the old events havent been updated to use getters.
We are however changing to use getters. So anything new should get them.

Reply:
Gotcha. Fixed things again; being distracted means I forgot to change the usage portion in the test mod.